data/material-model/entropy-table/opxtable/interpolate.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/python
+
+# This interpolation script takes the HeFESTo table in its original format,
+# subsamples it, reorders the columns, and adds a header so that it can be used
+# by the AsciiDataLookup class inside ASPECT.
+
+import numpy as np
+
+input_filename = "opxtable_s.aspect.full"
+output_filename = "opxtable_s.aspect"
+
+data = np.genfromtxt(input_filename)
+table = data.reshape(1401,241,9)
```

Comment:
It would also be nice to know how one would have to change these numbers if someone wants to convert their own HeFESTo table.